en/deploy/README.md

```diff
@@ -200,7 +200,7 @@ You can also go to the "Files" tab and navigate around using PythonAnywhere's bu
 Your site should now be live on the public Internet!  Click through to the PythonAnywhere "Web" tab to get a link to it.  You can share this with anyone you want :)
 
 
-> **Note** This is a beginners' tutorial, and in deploying this site we've taken a few shortcuts which aren't ideal from a security point of view.  If and when you decide to build on this project, or start a new project, you should review the [Django deployment checklist](https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/) for some tips on securing your site.
+> **Note** This is a beginners' tutorial, and in deploying this site we've taken a few shortcuts which aren't ideal from a security point of view.  If and when you decide to build on this project, or start a new project, you should review the [Django deployment checklist](https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/) for some tips on securing your site.
```

Comment:
Should we parametrize the version in docs.djangoproject.com URLs, too? (Maybe not, as changed URLs need to be checked anyway for whether they still work.)

Reply:
I think it is best to do it manually to check if the URLs are still valid. It is easy to grep all of them when Django 2.1 come out ;)